src/main/java/com/epam/tc/hw5/fragments/HeaderMenu.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.tc.hw5.fragments;
+
+import com.epam.tc.hw5.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class HeaderMenu extends BasePage {
+
+    @FindBy(xpath = "//li[@class='dropdown']")
+    private WebElement serviceMenuDropDown;
+
+    @FindBy(xpath = "//a[text()='Different elements']")
+    private WebElement differentElementsServiceElement;
+
+    @FindBy(xpath = "//a[text()='User Table ']")
+    private WebElement userTablePageElement;
+
+    public HeaderMenu(WebDriver driver) {
+        super(driver);
+    }
+
+    public void clickServiceMenuAndThenDifferentElementService() {
+        serviceMenuDropDown.click();
+        differentElementsServiceElement.click();
```

Comment:
I don't love this very long method name... clickServiceMenuAndThenDifferentElementService()

What do you want to do? Open another page? Or smth else?

Please, choose better name)

Reply:
The method opens the 'Different Element' page. The method has been renamed to openDifferentElementPage()